src/jsonMigrationScript130.ts

```diff
@@ -0,0 +1,138 @@
+import fs from 'fs';
```

Comment:
This is the migration script I wrote. I'll keep it here until the PR gets merged in case any new networks get added.

Reply:
I was going to suggest to keep it in `scripts` directory so that the migration can be replayed in the future.

Reply:
Why do we want to keep it? I don't think we need to use it after the PR merged.